README.md

```diff
@@ -2,42 +2,314 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/sodascience/metasynth/HEAD?labpath=examples%2Fgetting_started.ipynb)
 [![Google Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sodascience/metasynth/blob/main/examples/getting_started.ipynb)
 [![docs](https://readthedocs.org/projects/metasynth/badge/?version=latest)](https://metasynth.readthedocs.io/en/latest/index.html)
-
 # MetaSynth
+MetaSynth is a powerful Python package designed for generating synthetic data, primarily aimed at facilitating rigorous code testing and ensuring reproducibility. 
 
-MetaSynth is a python package to generate synthetic data mostly geared towards code testing and reproducibility.
-Using the [ONS methodology](https://www.ons.gov.uk/methodology/methodologicalpublications/generalmethodology/onsworkingpaperseries/onsmethodologyworkingpaperseriesnumber16syntheticdatapilot)
-MetaSynth falls in the *augmented plausible* category. To generate synthetic data, MetaSynth converts a polars DataFrame
-into a datastructure following the [GMF](https://github.com/sodascience/generative_metadata_format) standard file format.
-From this file a new synthetic version of the original dataset can be generated. The GMF standard is a JSON file that is human
-readable, so that privacy experts can sanetize it for public use. 
+The core aim of MetaSynth is to enable researchers and data owners to generate and share a synthetic rendition of their sensitive tabular data while preserving privacy. By reducing the level of statistical information in our models compared to existing synthetic data solutions, MetaSynth ensures privacy and mitigates concerns over data misuse. This provides data owners with a FAIR (Findable, Accessible, Interoperable, Reusable) approach to data sharing. For researchers, it makes the initiation of work with sensitive data easier and increases the reproducibility of their research.
 
+The primary functionalities of MetaSynth are **metadata generation** from a dataset and **synthetic data generation** based on the derived metadata. The distinctive advantage is the separation of synthetic data crea
 
 ## Features
+### Generating metadata from a dataset
+MetaSynth can generate metadata from any given dataset (provided as polars or pandas dataframe). 
+
+This metadata describes the structure and characteristics of the variables in the synthetic dataset (including their names, types, data types, proportion of missing values, and distribution specifications). 
+
+This metadata follows the GMF standard,  [Generative Metadata Format (GMF)](https://github.com/sodascience/generative_metadata_format) and as such is designed to be easy to read. It can be exported as .JSON file allowing for manual and automatic editing, as well as easy sharing.
+
+![image](https://github.com/Samuwhale/metasynth/assets/22395533/6aadf296-f29e-4d55-a53e-fd98c15dc5ee)
```

Comment:
yeah I already was thinking it's a bit hacky, but for the user it probably makes no difference hence why I went for this quick approach to quickly be able to test images. 

I'll add the images directly to the repo (in the readthedocs folder, so they can be used there later too), so that they can be linked :)